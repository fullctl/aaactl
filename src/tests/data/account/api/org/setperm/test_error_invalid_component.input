{
  "client": "api_client",
  "data": {
    "id": 4,
    "component": "unknown",
    "permissions": "crud"
  }
}
