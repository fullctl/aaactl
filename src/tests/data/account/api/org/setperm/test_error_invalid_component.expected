{
  "status": 400,
  "response": {"data": [], "errors": {"component": ["Not a valid permissioning component"]}, "profiling": {"queries": 0}}
}
